packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -52,7 +52,7 @@ export const Query = {
 
     const after = maybeAfter ? Number(maybeAfter) : 0
     const searchArgs = {
-      page: Math.ceil(after / first),
+      page: Math.ceil(after / first!),
```

Comment:
`first` now can indeed be `undefined`, right? Since it's not required anymore in the schema. In that case, `page` here would also be `undefined`.

Why is this non-null assertion necessary?

Reply:
this code is not from that PR, I won't cover it here, so we can open another PR to solve this, what do you think?